openlibrary/templates/lib/header_dropdown.html

```diff
@@ -33,7 +33,7 @@
             $(props['label'])
             <span class="shift">$_('Menu')</span>
 
-        <img class="down-arrow" aria-hidden="true" src="/static/images/down-arrow.png" alt="" role="presentation" width="7" height="4">
+        <img class="down-arrow" aria-hidden="true" src="/static/images/down-arrow.png" alt="$_('down-arrow')" role="presentation" width="7" height="4">
```

Comment:
For accessibility, an image that is `role="presentation"` basically means it has no function purpose and its purely aesthetic, so should be ignored by screen readers. So we don't want this image to have an alt text.

```suggestion
        <img class="down-arrow" aria-hidden="true" src="/static/images/down-arrow.png" alt="" role="presentation" width="7" height="4">
```